src/main/resources/config/liquibase/changelog/20250521014500_changelog.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="
+            http://www.liquibase.org/xml/ns/dbchangelog
+            https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.4.xsd">
+
+    <changeSet id="20250521014500-add-jobid-and-status-to-lecture-transcription" author="goyal">
+        <addColumn tableName="lecture_transcription">
+            <column name="job_id" type="VARCHAR(255)">
+                <constraints nullable="true" unique="true"/>
+            </column>
+            <column name="transcription_status" type="ENUM('PENDING','PROCESSING','COMPLETED','FAILED')" defaultValue="PENDING">
```

Comment:
Please adapt this to be compatible with Postgres.

See https://github.com/ls1intum/Artemis/blob/cf102a3cd97bdc2871297544273086cdff2cfd3e/docs/dev/guidelines/database.rst?plain=1#L30 for more information.